src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -102,6 +108,11 @@ export class AssetsTable extends React.Component {
     return this.trashcanRefs[focusAsset.id];
   }
 
+  getImageThumbnail(thumbnail) {
+    const baseUrl = this.props.courseDetails.base_url ? this.props.courseDetails.base_url : '';
```

Comment:
Nit: What do you think about using [default props](https://reactjs.org/docs/react-component.html#defaultprops) instead of the ternary?